### DISCUSSION TITLE
SofaPyton3 plugin

### DISCUSSION THREAD

**Discussion by AngelicaScamarcia** [sofa-framework/sofa#3845] [Build / Config / Environment] [answered]
Hi there,
I have the source version of SOFA v22.12.00, VS 2022 and I'm struggling with the SofaPython3 plugin. 
In particular, I followed the step on the documentation:
•	Download Python 3.8.10
•       Copy the file plugin_list.conf.default in <SOFA_build>/lib, renaming it plugin_list.conf and adding the line SofaPython3 NO_VERSION at the beginning.
 
Then I set the variable as:
SOFA_ROOT = C:/Program Files/sofa/build
PYTHON_ROOT = C:/Users/scama/AppData/Local/Programs/Python/Python38
PYTHONPATH = C:/Users/scama/AppData/Local/Programs/Python/Python38/Lib/site-packages
 
I edited the system variable Path as:
 
![image](https://github.com/sofa-framework/sofa/assets/95097437/2d3abaec-6057-488f-8f72-77c1ed45995f)

 
Despite these changes, I still have the error when I run INSTALL on VS.
 
![image](https://github.com/sofa-framework/sofa/assets/95097437/4f703ff5-5aa6-4a25-80aa-11fac58b659c)
 
 
Am I doing something wrong? What can I do?

**Comment by alxbilger** [comment]
By curiosity, why do you need to run INSTALL?

I tested and I don't have the error.

**Reply by AngelicaScamarcia** [reply]
Can I use SofaPython3 just compiling it?

**Comment by alxbilger** [comment, marked as answer]
Run VS with administrator privileges

**Reply by AngelicaScamarcia** [reply]
It works, thank you so much!
